### ISSUE TITLE
[BUG][NOWEB] - Sender presence keeps on typing after sending message.

### ISSUE THREAD

**Issue by augustovmedeiros** [allburov/whatsapp-http-api#379]
After the sender sends a message, the sender presence gets stuck on "typing", while he doesnt send the message the status will change after stop "typing" to "online" as it should
**Version**

```json
{
  "version": "2024.6.5",
  "engine": "NOWEB",
  "tier": "PLUS",
  "browser": "/usr/bin/google-chrome-stable"
}
```

**To Reproduce** Steps to reproduce the behavior:

1. Open a NOWEB session
2. Send a message to the session
3. Subscribe for sender presence with http://ip:3000/api/session/presence/id_number@c.us/subscribe
4. Check the status with http://ip:3000/api/session/presence/id_number@c.us
5. See error

https://github.com/devlikeapro/waha/assets/33730078/e41aba15-5914-4722-94ad-4b0e68cb4fed

**Expected behavior**

> It should change to "online" closely after recieving the message and the sender has stopped typing.

**Engine** I'm using **NOWEB** engine.

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
oh, my bad, I thought it was about the sender https://github.com/devlikeapro/waha/issues/365
Checking it!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Fixed in `2024.6.6`!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
